whois-commons/src/main/java/net/ripe/db/whois/common/DatabaseDummifierJmx.java

```diff
@@ -242,6 +252,14 @@ public static void main(String[] argv) {
         String jdbcUrl = options.valueOf(ARG_JDBCURL).toString();
         String user = options.valueOf(ARG_USER).toString();
         String pass = options.valueOf(ARG_PASS).toString();
-        new DatabaseDummifierJmx().dummify(jdbcUrl, user, pass);
+        Environment env;
+        try {
+            env = Environment.valueOf(options.valueOf(ARG_ENV).toString().toUpperCase());
+        } catch (IllegalArgumentException ex){
```

Comment:
you can also use EnumUtils.getEnumIgnoreCase, feel free to ignore

Reply:
I can see that method in EnumUtils :S